docs/Users_Guide/working_with_config.rst

```diff
@@ -161,3 +161,16 @@ will print the following to the stdout::
   Keys that are set to empty:
       FV3GFS.nomads.file_names.nemsio
       FV3GFS.nomads.testempty
+
+.. _conf_conversion:
+
+^^^^^^^^^^^^^^^^
+Conversion flags
+^^^^^^^^^^^^^^^^
+
+``set_config.py`` has three flags that will convert given input, config, or outfile to the user-provided file type::
+    --input_file_type
+    --config_file_type
+    --outfile_file_type
+
+They accept YAML ('YAML'), bash/ini ('INI'), or namelist ('F90') file types.
```

Comment:
We may want to include some of the caveats we've included in the code...`input_file_type` and `config_file_type` can be provided as YAML, INI, and F90, but `output_file_type` can also be `FieldTable`, for example.

The tool also requires that all three files are compatible structures (depth), and it might be helpful to highlight that here.